openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -118,13 +118,12 @@ Deferred Credential Issuance:
 This specification defines an API for Credential issuance provided by a Credential Issuer. The API is comprised of the following endpoints:
 
 * A mandatory Credential Endpoint from which Credentials can be issued (see (#credential-endpoint)).
```

Comment:
```suggestion
* A mandatory Credential Endpoint from which Credentials can be issued (see (#credential-endpoint)). From this endpoint, one Credential, or multiple copies of the same Credential Dataset can be issued in one request.
```

Reply:
I would avoid the term "copy", since I don't think it is used elsewhere in the document.

Reply:
Changed it to `multiple Credentials with the same Credential Dataset can be issued in one request`